napari/utils/_tests/test_theme.py

```diff
@@ -98,6 +101,11 @@ def test_rebuild_theme_settings():
     settings.appearance.theme = "another-theme"
 
 
+@pytest.mark.skip(
+    os.getenv('CI', '0') != '0' and sys.version_info < (3, 9),
```

Comment:
Ok this condition was wrong and it always skips.
Not sure why as it's the same as in `napari/_tests/utils.py`
```suggestion
    os.getenv('CI') and sys.version_info < (3, 9),
```